ReleaseHistory.md

```diff
@@ -1,4 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+## UNRELEASED
+* DEP: Add explicit package references to `Sarif` and `Sarif.Driver` to resolve version conflict build error.
+  `System.Diagnostics.Debug` 4.3.0,
+  `System.IO.FileSystem.Primitives` 4.3.0
+  `System.Text.Encoding.Extensions` 4.3.0
+
```

Comment:
Add period to end of line to be consistent with other DEP release notes that include a version list.